atlasdb-commons/src/test/java/com/palantir/remoting/HeaderAccessUtilsTest.java

```diff
@@ -97,6 +101,15 @@ public void caseInsensitiveGetShortcircuits() {
         assertEquals(VALUE_2, HeaderAccessUtils.shortcircuitingCaseInsensitiveGet(testMap, KEY_2.toUpperCase()));
     }
 
+    @Test
+    public void parseContentTypePlainText() {
+        Map<String, Collection<String>> testMap = Maps.newLinkedHashMap();
+        testMap.put("Content-Type", ImmutableList.of("text/plain;charset=iso-8859-1"));
```

Comment:
nits, if we were going to keep this test:
* We imported HttpHeaders.CONTENT_TYPE, but still use the magic string "Content-Type" here
* Normal var naming convention in the next line is that all caps = static vars, and we should avoid naming a var the same thing as a static variable we're using